.github/workflows/containers.yml

```
@@ -0,0 +1,203 @@
+        if: hashFiles(format('{0}/{1}', matrix.path, 'test.sh')) != ''
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.YELLDUCK_GIHUB_PKG_TOKEN }}
+
+  build-and-publish-docker-image:
+    name: Build & Push Container
+    needs: [set-matrices, test]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{fromJson(needs.set-matrices.outputs.build-matrix)}}
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)